lib/manageiq/appliance_console/message_configuration_client.rb

```diff
@@ -0,0 +1,118 @@
+require "awesome_spawn"
+require "fileutils"
+require "linux_admin"
+require 'net/scp'
+require "manageiq/appliance_console/message_configuration"
+
+module ManageIQ
+  module ApplianceConsole
+    class MessageClientConfiguration < MessageConfiguration
+      attr_reader :server_hostname, :server_username, :server_password, :installed_files
+
+      def initialize(options = {})
+        super(options)
+
+        @server_hostname, @server_username, @server_password = nil
```

Comment:
Is this needed since they're defined below?

Reply:
Good point. I also fixed my incorrect fix of unnecessary initialization to nil in commit: https://github.com/ManageIQ/manageiq-appliance_console/pull/130/commits/9406e5bfb02693d7a32e0f853f6ef9a3a889a21b